.github/CONTRIBUTING.md

```diff
@@ -33,7 +34,7 @@ All you need is [VSCode][2] and [NodeJS][0].
 
 ### Launch Configurations
 
-- _Run Extension_ will run the extension in a separate _vsocde instance_.
```

Comment:
funny

Reply:
it's your fault!
now that I know of _cspell_ I see all of my typos.
😁